src/main/java/se/sundsvall/disturbance/api/validation/OneOf.java

```diff
@@ -0,0 +1,25 @@
+package se.sundsvall.disturbance.api.validation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+
+@Documented
+@Target({ElementType.CONSTRUCTOR, ElementType.FIELD, ElementType.PARAMETER, ElementType.TYPE_USE})
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = OneOfConstraintValidator.class)
+public @interface OneOf {
```

Comment:
Could be useful in other services as well. Put this in dept44-validators?

Reply:
Yes, this is Jonas baby, I think he has some improvements as well.